js/data-analyst-agent/README.md

```diff
@@ -45,22 +46,31 @@ The Data Analyst Agent combines SQL querying capabilities with data visualizatio
 
 ### Running the Bot
 
-#### Option 1: Using Teams Toolkit (Recommended)
+#### Option 1: Local deployment using Teams Toolkit (Recommended)
 1. Open the project in Visual Studio Code
-2. Press F5 to start the debug session
+2. Press F5 to start the debug session (Debug Edge)
 3. Teams Toolkit will handle:
    - Starting the local bot server
    - Tunneling for external access
    - Opening Teams with your bot loaded
 
-#### Option 2: Manual Start
-1. Start the development server:
-   ```bash
-   npm run dev
-   ```
-2. The bot will be available on `http://localhost:3978`
-3. Use ngrok or similar to expose your local server
-4. Update your bot endpoint in Azure Bot Service
+
+#### Option 2: Deploy to Azure using Teams Toolkit
+1. Create an empty `.env.dev` file and place it in the `env` folder.
+2. Create a `.env.dev.user` file in the `env` folder and add the following contents to it:
```

Comment:
Should we still push the /env folder up so that devs just need to fill it in?

Reply:
Good idea I'll create a Fast follow PR for that